packages/@lwc/shared/src/api-version.ts

```diff
@@ -80,6 +80,11 @@ export const enum APIFeature {
      * instead of comment nodes.
      */
     USE_COMMENTS_FOR_FRAGMENT_BOOKENDS,
+    /**
+     * If enabled, allows slot forwarding for light DOM slots. This will cause the slot attribute of the slotted
+     * content to be updated to match the slot attribute of the light DOM slot it slotted into.
+     */
```

Comment:
Not sure if this is descriptive enough but open to suggestions!

Reply:
We will want to have more detailed examples in the release notes (e.g. see the [v4.0.0 release notes](https://github.com/salesforce/lwc/releases/tag/v4.0.0)). But this is great for just the code comment here!

BTW, now would be a good time to draft those release notes. Could you put them in the final PR (#3883) under the "observable changes" section so we have them ready when we do the release?

Reply:
Will do!